### DISCUSSION TITLE
Discord

### DISCUSSION THREAD

**Discussion by Mdbgamer** [BlasterM5/ponyorigins#21] [General]
With both a rather impressively indepth Origins addon and an entire pack built around it, have you considered a Discord server where people who enjoy the mod/pack can hang out? If one already exists, where is it so my blind arse can find it? :D

**Comment by Mdbgamer** [comment]
Yaknow, I hadn't thought of the name of this thread at the time, but that could be misleading... ah well. x)

**Comment by BlasterM5** [comment]
I have, however, I prefer the github environment.

**Reply by Mdbgamer** [reply]
That's fair!

**Comment by Mdbgamer** [comment]
I do have another question: When it comes to being the first to "Complete a difficult advancement" in the server, does that make going for Alicornhood even worth it if you're a late join to a server where all the difficult advancements have been obtained? I read that in the Alicorn section of the wiki, and have no idea if this is outdated information or not, but it _seems_ like once all the "difficult advancements"(stuff like beating the dragon, making a beacon, getting full netherite) are taken by other players, that leaves any others that wanted to go for Alicornhood out of the running entirely. I could be mistaking something though.

**Reply by BlasterM5** [reply]
Go for Chaos - there's actually two "always open" but considerably hard advancements that allow you to eventually get Alicorn. It's setup like this because Alicorn is supposed to be something difficult to achieve for the average player, and limits the numbers of Alicorns on the server to those that really want to achieve it, given they are more powerful than the standard races by design.

To become an Alicorn, you must go above and beyond the norm.

**Reply by Mdbgamer** [reply]
That's fair, honestly. Would modded ones count? Or are only the vanilla ones taken into account?

**Reply by BlasterM5** [reply]
There are a few modded advancements that count - from Gaia, Ad Astra, and Chef's Delight mainly.

**Comment by BlasterM5** [comment]
I do in fact have an R&D Discord now. Link's on the modrinth page

**Reply by Mdbgamer** [reply]
Awesome. Will check that out. :D
